src/CommunityToolkit.Maui.Core/Essentials/FileSaver/FileSaverImplementation.shared.cs

```diff
@@ -8,7 +10,7 @@ public async Task<FileSaverResult> SaveAsync(string initialPath, string fileName
 		try
 		{
 			cancellationToken.ThrowIfCancellationRequested();
-			var path = await InternalSaveAsync(initialPath, fileName, stream, cancellationToken);
+			var path = await InternalSaveAsync(initialPath, fileName, stream, new Progress<double>(), cancellationToken);
```

Comment:
if the user doesn't provide an `IProgress` object we can cache this one and reuse it on all calls? At the end will not affect the user since they choose to not follow the progress.